docs/source/quickstart.rst

```diff
@@ -0,0 +1,37 @@
+.. _getting-started-chapter:
+
+###############
+Getting Started
+###############
+
+****************
+Buying a GoPiGo3
+****************
+
+For buying a `GoPiGo3`_ robot, please head over to our online `shop`_ and search for the `GoPiGo3`_ robot. And don't forget to get some sensors - with them, everything is a lot more fun.
```

Comment:
Take out "And don't forget to get some sensors - with them, everything is a lot more fun."  -- Replace with something a little more formal:  "You can also add sensors to your robot such as x, y, and z."